lib/ansible/module_utils/facts.py

```diff
@@ -604,6 +604,10 @@ class Distribution(object):
     This is unit tested. Please extend the tests to cover all distributions if you have them available.
```

Comment:
please have a look at the existing tests, It's straight-forward to extend them to other distributions. See also the discussion on the previous issues for how to do this. Let me know if you have specific questions.